src/Map.js

```diff
@@ -67,7 +67,9 @@ export default class Map {
 
   /** *************************** static *************************** */
   static formatClusterText(count) {
-    if (count > 1000) {
+    if (count > 1000000) {
```

Comment:
@iib0011 you could change this to `if (count >= 1000000)` so that it would display 1.0M instead of 1000K

Reply:
.tofixed(1) shows 1 digit like 1.1M

Reply:
@iib0011 yes that's correct, but i meant for the case when the `count` is exactly equals to 1000000, because for this case line 73 will run instead of line 71